src/main/commandServer/__tests__/settingsValidator.spec.ts

```diff
@@ -0,0 +1,150 @@
+import _ from 'lodash';
+import SettingsValidator from '../settingsValidator';
+import * as settings from '@/config/settings';
+import { RecursivePartial } from '~/utils/recursivePartialType';
+
+const cfg = settings.init();
+cfg.kubernetes.version ||= '1.23.4';
```

Comment:
I assume older versions of node we still support don't have the `||=` operator, because I never saw any lint warnings here on in CI.

Reply:
All this goes through the TypeScript compiler (to go from `*.ts` to `*.js`), which takes care of polyfills for stuff like that. 

Also, [devmo](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment#browser_compatibility) shows it as supported since Node 15 (and we're using Node 16 at minimum)

---
Anyway, the lint warning is about new line after `const`…